{
  control.expectAndThrow(mock.objectReturningMethod(4),EXCEPTION,MockControl.ONE);
  control.replay();
  try {
    mock.objectReturningMethod(4);
    fail();
  }
 catch (  RuntimeException exception) {
    assertSame(EXCEPTION,exception);
  }
  control.verify();
}
